lte/gateway/docker/s1ap/docker-compose.override.yaml

```diff
@@ -0,0 +1,13 @@
+---
+version: "3.7"
+
+services:
+  s1aptester:
+    build:
+      context: ${BUILD_CONTEXT}
```

Comment:
how is `BUILD_CONTEXT` set? I do not see any relevant code in [lte/gateway/docker/s1ap/build-s1ap.sh](https://github.com/magma/magma/pull/11308/files#diff-c7db871d0933fb35d62808ce153a93ef0cdac17e52bee7f7ba2bdeec51cd67a2)

Reply:
Was this clarified?

Reply:
I tested the script and it successfully builds the images